.github/workflows/regression_test_summariseEpoch.yml

```diff
@@ -0,0 +1,43 @@
+name: Python - summariseEpoch.py
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+   # - name: Lint with flake8
+   #   run: |
+   #     # stop the build if there are Python syntax errors or undefined names
+   #     flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+   #     # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+   #     flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Compile Parser
+      run: javac -cp java/JTransforms-3.1-with-dependencies.jar java/*.java
+    - name: Load data
+      run: bash utilities/downloadDataModels.sh
+    - name: Process data using summariseEpoch.py
+      run: python accelerometer/summariseEpoch.py data/sample.cwa
```

Comment:
I am very confused here. Shouldn't we have the epoch as input instead of ``sample.cwa`` ? 

Also I guess what we really want to do here is to parse the `sample.cwa` by `python3 accProcess.py data/sample.cwa.gz`? Then compare the output epoch and summary files?

Reply:
yes it was meant to be accProcess.py, oops! Thanks for spotting this.